File: informe 1/6.cpp
#include <iostream>
using namespace std;
int main() {
    int horas, minutos, segundos;
    int totalSegundos;
    cout << "Ingrese las horas: ";
    cin >> horas;
    cout << "Ingrese los minutos: ";
    cin >> minutos;
    cout << "Ingrese los segundos: ";
    cin >> segundos;
    totalSegundos = (horas * 3600) + (minutos * 60) + segundos;
    cout << "El equivalente en segundos es: " << totalSegundos << endl;
    return 0;
}

File: informe 1/6.py
def convertir_a_segundos(horas, minutos, segundos):
    return horas * 3600 + minutos * 60 + segundos

h = int(input("Ingrese las horas: "))
m = int(input("Ingrese los minutos: "))
s = int(input("Ingrese los segundos: "))

print("El equivalente en segundos es:", convertir_a_segundos(h,m,s))